setuptools/command/test.py

```diff
@@ -229,6 +229,7 @@ def run_tests(self):
             None, None, self._argv,
             testLoader=self._resolve_as_ep(self.test_loader),
             testRunner=self._resolve_as_ep(self.test_runner),
+            exit=False,
```

Comment:
This parameter doesn't exist on Python 2.6. You'll need to handle the case where it doesn't exist or just check before passing it.